index.js

```diff
@@ -129,11 +129,13 @@ contestPool
     .bufferBy(kefir.interval(ONE_DAY / config.tweetsPerDay))
     .onValue(tweets => {
         var mostRetweets = tweets.reduce((mostRetweeted, tweet) => {
-            return tweet.retweet_count > mostRetweeted.retweet_count ?
+            return (mostRetweeted === null || tweet.retweet_count > mostRetweeted.retweet_count) ?
```

Comment:
I think this would make more sense as 

```js
if (tweets.length > 0) {
    var mostRetweets = tweets.reduce((mostRetweeted, tweet) => {
            return tweet.retweet_count > mostRetweeted.retweet_count ?
                tweet : mostRetweeted;
        });

        enterContest(mostRetweets);
}
```

Providing the initial result as a null and then handling that special case seems a little awkward to me.